Prototope/Layer.swift

```diff
@@ -820,3 +1007,47 @@ private func incorporateTouches(touches: NSSet, intoTouchSequenceMappings mappin
 	let updatedTouchSequences = incorporateTouchSequences(touchSequencesFromTouchSet(touches), intoTouchSequenceMappings: mappings)
 	return touchSequenceMappingsFromTouchSequences(updatedTouchSequences)
 }
+
+#endif
+
+
+#if os(iOS)
+	import UIKit
+	public typealias SystemImageView = UIImageView
+	#else
+	import AppKit
+	public typealias SystemImageView = NSImageView
+	
+	extension SystemView {
+		func setNeedsLayout() {
+			// TODO(jb): What's the OS X equiv of this again?
```

Comment:
Oh, wait, no, that's `CALayer` that has no argument. I'm confused now because Dash does claim that `NSView` has such a method (introduced in 10.7) but developer.apple.com doesn't??